### DISCUSSION TITLE
authorize device via email throw error

### DISCUSSION THREAD

**Discussion by shanei5107** [getlantern/lantern-forum-cn#694] [Q&A] [answered]
<img width="351" alt="image" src="https://github.com/getlantern/lantern-forum-cn/assets/3353784/503222ef-8e72-4512-9944-083abcd8e38c">

**Comment by shanei5107** [comment]
macOS: Sonoma 14.5

**Comment by MaxMKMax** [comment, marked as answer]
@shanei5107 Hello, thank you for your feedback. We are fixing issues and making optimizations. Please wait. We are very sorry for affecting your experience. Thank you for your waiting and support.

Can you help us by reporting your issue within the app? You can access the Lantern application and submit log/reports.

go to Support
select Report an Issue
choose issue type
include additional details
hit Send Report (click to send report)

When you submit a problem report in the app, you will receive an email. There is an 8-digit number in the email title. Please come back here and leave your log/ticket number. Because I need your feedback report number to communicate with the technology team about your problem. problem. Otherwise I would not be able to submit the problem you encountered to the technology team. Sorry, I need your help. Thank you!

With this information, our technology team can solve the problem as quickly as possible. You can let me know in the message area. Thanks.

**Reply by shanei5107** [reply]
already feedback issue within the app

**Reply by MaxMKMax** [reply]
@shanei5107 can you provide me the 8 digit in app report log number as well? Thanks.

**Comment by shanei5107** [comment]
reinstall new version, it works

**Reply by MaxMKMax** [reply]
Okay, thank you so much.
